packages/medusa/src/api/admin/product-categories/validators.ts

```diff
@@ -74,6 +79,10 @@ export const AdminUpdateProductCategory = z
   })
   .strict()
 
+export const AdminUpdateProductCategory = WithAdditionalData(
```

Comment:
todo: categoriesUpdated does not pass `additional_data` into the hook. You can either:

1. revert this change
2. update `packages/core/core-flows/src/product-category/workflows/update-product-categories.ts` similar to what you did for create product categories